### ISSUE TITLE
Synch JSON structure with our aubrey resource object

### ISSUE THREAD

**Issue by ldko** [unt-libraries/aubrey-transcription#3]
Now that we are looking at putting the JSON data served by aubrey-transcription into Aubrey resource objects at a manifestation's file set level, we should make the data consistent for easier assimilation.

1. For keys it looks like `MIMETYPE` and `USE` should be all caps and `file_size` should be renamed to `SIZE`.

2. We should look at converting the list to a dictionary structure where looking up data is by the manifestation and file set number.

**Comment by somexpert** [comment]
Should be easy enough. I'll get part one up in a branch here in a minute. Part 2 will probably be a separate PR.

**Comment by vphill** [comment]
What about a structure like this that would allow for 

`data[manifestation][fileSet]`

and then start looping through the list of transcription files for a given manifestation/fileSet. 

```json
{
   "1":{
      "1":[
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-captions-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         },
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-captions-spa.vtt",
            "language":"spa",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         },
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-subtitles-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"subtitles"
         }
      ],
      "2":[
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_2-captions-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         }
      ]
   }
}
```

**Comment by somexpert** [comment]
Looks good to me, I can make it happen.

**Comment by somexpert** [comment]
For the manifestation key, do you want just the number, or the "m" that goes with it too?

**Comment by somexpert** [comment]
Getting rid of the 'm'.

**Comment by somexpert** [comment]
Here is the example @vphill gave:

What about a structure like this that would allow for 

`data[manifestation][fileSet]`

and then start looping through the list of transcription files for a given manifestation/fileSet. 

```json
{
   "1":{
      "1":[
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-captions-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         },
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-captions-spa.vtt",
            "language":"spa",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         },
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_1-subtitles-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"subtitles"
         }
      ],
      "2":[
         {
            "file_size":0,
            "flocat":"http://example.com/me/ta/te/st/1/metatest1/metatest1_m1_2-captions-eng.vtt",
            "language":"eng",
            "mimetype":"text/vtt",
            "use":"vtt",
            "vtt_kind":"captions"
         }
      ]
   }
}
```

**Comment by somexpert** [comment]
This was addressed in #5
